main_routers/workshop_router.py

```
@@ -709,48 +815,225 @@ async def unsubscribe_workshop_item(request: Request):
+                        logger.info(f"已保存更新后的characters.json，删除了 {deleted_count} 个角色卡")
+                        
+                        # 重新加载配置
+                        try:
+                            initialize_character_data = get_initialize_character_data()
+                            if initialize_character_data:
+                                # 尝试获取事件循环并安全地调用异步函数
+                                try:
+                                    loop = asyncio.get_event_loop()
+                                    if loop.is_running():
+                                        # 如果事件循环正在运行，使用create_task
+                                        loop.create_task(initialize_character_data())
```

Ruff (0.14.10)

Store a reference to the return value of `loop.create_task`

(RUF006)